wallet-new/server/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -229,6 +229,9 @@ instance ToDocs Account where
 instance ToDocs AccountUpdate where
   descriptionFor _ = updateDescr (Proxy @Account)
 
+instance ToDocs AddressValidity where
+  descriptionFor _ = "Whether an address is valid or not."
```

Comment:
What about documenting the criteria needed for an address to be considered valid? The next exchange integrator will thank you 😀